apps/server/src/api/controllers/suppliers/list-suppliers-controller.ts

```diff
@@ -0,0 +1,26 @@
+import type { IHttp } from '@stocker/core/interfaces'
+import { ListSuplliersUseCase } from '@stocker/core/use-cases'
+import { HTTP_STATUS_CODE } from '@stocker/core/constants'
+
+import { suppliersRepository } from '@/database'
+
+type RouteParams = {
+  page: string
+  companyId: string
+}
+
+export class ListSuppliersController {
+  async handle(http: IHttp) {
+    const { companyId } = await http.getUser()
+    const { page } = http.getQueryParams<RouteParams>()
+    if (!companyId) {
+      return http.send(null, HTTP_STATUS_CODE.badRequest)
+    }
+    const pageNumber = parseInt(page || '1', 10)
```

Comment:
`parseInt` function is no longer used by the community. Prefer to use `Number` constructor instead.